.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "semi": true,
+  "trailingComma": "es5",
+  "singleQuote": true,
+  "printWidth": 100,
+  "tabWidth": 2
+}
```

Comment:
Do we want to keep it consistent with the rest of projects like [Cavalry](https://github.com/x-team/xp-cavalry/blob/main/.prettierrc)?